src/coffee/client/modules/todo/todo_item_module.coffee

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2015 by Looker Data Services, Inc.
+# All rights reserved.
+#
+
+angular   = require 'angular'
+templates = require '../../templates'
+{EVENT}   = require '../../../constants'
+_         = require '../../../underscore'
+
+############################################################################################################
+
+### ex: TodoItem flow
+1. User clicks the submit button, `$scope.addTodoItem()` fires TodoItems's addTodoItem action
+2. `onAddItem()` runs and creates a new ReadOnlyView TodoItem, pushes it into @_items
+    (3. Register doItem action to run when the 'done' field of the ReadOnlyView is changed)
+4. TodoItemStore sends a notification that something has been added and changed
+
+############################################################################################################
+
+todo = angular.module('todo', ['reflux'])
+
+todo.factory 'TodoItemStore', (Todo, TodoItemActions, TextInputStore, ReadOnlyView, reflux) ->
+    reflux.createStore
+
+        listenables: TodoItemActions
+
+        init: ->
+            @_items = []
+
+        onAddItem: -> # (2)
+            @_todo = new ReadOnlyView({
```

Comment:
I think that's mostly a result of this app being very simple.  If you had multiple users, the item store might contain all items for all users, while the list store might keep track of just the items belonging to the current user.  As things stand, you could probably dispense with the list store altogether.